[feature][bugfix]  support all possible configurations in pkg.conf(5)

trombik opened the pull request:
this PR fixes #5

also, 

* refactor tests
* fix wrong variable name
* create configuration files before installing pkg

Commit:
QA

Commit:
support all posiible configurations in pkg.conf(5)

* refactor tests

Commit:
fix wrong variable name in README

Commit:
rubocopfy

mitsururike reviewed (commented):
* typo: s/hos tmust/host must/ in README.md
* the role should fail first in `tasks/main.yml` as the host must be FreeBSD.

Commit:
failout when ansible_os_family is not FreeBSD

also

* fix a typo

https://github.com/reallyenglish/ansible-role-pkg/pull/6#pullrequestreview-69796748
